04_Intro_to_Butler.ipynb

```diff
@@ -122,15 +122,13 @@
    "source": [
     "#### 2.1 Butler registry and collections\n",
     "\n",
-    "The registry is a database containing information about available data products.\n",
-    "The registry helps the user to examine what collections of data products exist.\n",
+    "The database side of a data repository is called a registry.\n",
+    "The registry contains entries for all data products, and organizes them by _collection_, _dataset type_, and _data ID_.\n",
     "Use the registry to investigate a repository by listing all collections.\n",
     "\n",
-    "Find more about the registry schema [here](https://dmtn-073.lsst.io/).\n",
-    "\n",
```

Comment:
I dropped the reference to DMTN-073 because I'm sure it's horribly out of date; I don't think I've so much as looked at it for two years.